remsfal-core/src/main/java/de/remsfal/core/json/NodeDataJson.java

```diff
@@ -0,0 +1,27 @@
+package de.remsfal.core.json;
+
+import org.immutables.value.Value;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+@Value.Immutable
+@Schema(description = "Encapsulated data of a project tree node")
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public interface NodeDataJson {
+
+    @Schema(description = "Type of the node (e.g., 'property', 'building')", required = true, example = "property")
```

Comment:
Sonar tells me that this is deprecated. According to my brief research the example-parameter in OpenAPI-schemas could easily be replaced with `examples = {"..."}`. Do you think it might be sensible to change that?

Reply:
I have addressed the Issue in my latest merge request.